src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionVisibilityOptions.tag

```diff
@@ -12,8 +12,9 @@
 <c:set var="FEEDBACK_STUDENTS"><%=FeedbackParticipantType.STUDENTS.name()%></c:set>
 <c:set var="FEEDBACK_INSTRUCTORS"><%=FeedbackParticipantType.INSTRUCTORS.name()%></c:set>
 
-<div class="col-sm-12 margin-bottom-15px padding-15px background-color-light-green">
+<div class="col-sm-12 margin-bottom-15px padding-15px <%= fqForm.isQuestionHasResponses() ? "alert alert-danger" : "background-color-light-green" %>">
     <div class="margin-bottom-7px">
+        <c:if test="${fqForm.questionHasResponses}"><h4>Changing the visibility after collecting responses is not recommended.</h4></c:if>
```

Comment:
Done @damithc. This is what it looks like now:

<img width="1123" alt="ogy" src="https://user-images.githubusercontent.com/18088721/27508220-126aa980-58fe-11e7-9029-c509290aded8.png">

Justification for `<h4>`:

With `<h5>` it looks like this:
<img width="1116" alt="h5" src="https://user-images.githubusercontent.com/18088721/27508227-2d1e0164-58fe-11e7-9853-6313e1422c1a.png">

And with just a `<p>` it looks like this:
<img width="1124" alt="p" src="https://user-images.githubusercontent.com/18088721/27508233-3d5f55b4-58fe-11e7-8b6b-7920747788f6.png">

The `<h4>` & `<p>` combination is the only thing that, in my opinion, makes the message not seem out of place. Thoughts?